docs/source/_static/css/no_autoscroll.css

```diff
@@ -0,0 +1,3 @@
+pre {
```

Comment:
This might not work simply because you didn't add the path to this file to `html_css_files` in `conf.py`

Reply:
You might want to remove this file too